src/Flex.php

```diff
@@ -182,6 +182,27 @@ public function activate(Composer $composer, IOInterface $io)
         }
     }
 
+    private function getLockFilePath()
+    {
+        $composerFile = Factory::getComposerFile();
+        $lockFilePath = "json" === pathinfo($composerFile, PATHINFO_EXTENSION)
+                ? substr($composerFile, 0, -4).'lock'
+                :  $composerFile .  '.lock';
+        $symfonyLock = str_replace("composer.", "symfony.", $lockFilePath);
+        if (getenv("SYMFONY_LOCKFILE_PATH")) {
+            $symfonyLock  = getenv("SYMFONY_LOCKFILE_PATH");
+        }
+        return (object)[
+            "composer" => $lockFilePath,
+            "symfony" => $symfonyLock
+        ];
+    }
+
+    private function isComposerLockMissing()
+    {
+        return file_exists(Factor::getComposerFile()) && !file_exists($this->getLockFilePath()->composer);
```

Comment:
`Factor` is the wrong class name so this is broken

Reply:
🙄 OMG - thx